torch/onnx/_internal/exporter/_torchlib/ops/symops.py

```diff
@@ -9,10 +9,22 @@
 from onnxscript.onnx_opset import opset18 as op
 
 import torch
-from torch.onnx._internal.exporter._torchlib._tensor_typing import BOOL, IntType
+from torch.onnx._internal.exporter._torchlib._tensor_typing import (
+    BOOL,
+    FLOAT,
+    FloatType,
+    INT64,
+    IntType,
+)
 from torch.onnx._internal.exporter._torchlib._torchlib_registry import onnx_impl
 
 
+@onnx_impl(torch.sym_float, trace_only=True)
+def sym_float(self: FloatType | IntType) -> FLOAT:
```

Comment:
```suggestion
def sym_float(self: TensorType) -> FLOAT:
```

We can instead accept anything like so. I think python3.9 doesn't like this `|` syntax.